bucket/git-cola.json

```
@@ -0,0 +1,38 @@
+{
+    "version": "4.15.0",
+    "description": "A highly-caffeinated, powerful, and intuitive graphical user interface for Git",
+    "homepage": "https://git-cola.github.io/",
+    "license": "GPL-2.0-or-later",
```

```suggestion
    "license": "GPL-2.0-or-later",
    "suggest": {
        "git": "git"
    },
```

Looks like separate git installation should be [suggested](https://github.com/ScoopInstaller/Scoop/wiki/App-Manifests#optional-properties), as per
https://github.com/git-cola/git-cola?tab=readme-ov-file#windows

---

Makes sense